python/tvm/testing/aot.py

```diff
@@ -809,6 +810,8 @@ def run_and_check_body(base_path):
             + f" FVP_DIR={fvp_dir}"
             + custom_params
         )
+        print(os.path.join(codegen_path, "host", "src", "default_lib1.c"))
+        time.sleep(0)
```

Comment:
Is it really necessary?

Reply:
Oops, that's debugging code that should not have been staged. Fixed!